app/kubernetes/react/components/index.ts

```diff
@@ -197,6 +198,13 @@ export const ngModule = angular
       []
     )
   )
+  .component(
```

Comment:
can you put the definition inside `clusterManagementModule`?

Reply:
updated